src/main/scala/org/globalforestwatch/summarystats/afi/AFiAnalysis.scala

```diff
@@ -48,9 +48,9 @@ object AFiAnalysis extends SummaryAnalysis {
     val summaryDF = AFiAnalysis.aggregateResults(
         AFiDF
         .getFeatureDataFrame(summaryRDD, spark)
-        .filter(!$"gadm_id".contains("null"))
+        .filter(!$"gadm_id".substr(1, 3).contains("null"))
```

Comment:
I think we actually might not want to check if anything is null? "null" is valid for adm1 and adm2 depending on the country, the only thing we'd want to filter out is when iso is an empty string. 

Or maybe not filter anything at all? Because we saw there are occasionally results outside of GADM boundaries somehow, do we want to discard those or keep those?

Reply:
@jterry64 We currently use the row with `location_id = -1` and `gadm_id = ''` to hold summed values for the dissolved list. The filter disregards stray pixels (like over oceans) that would add another row with those specific values. Or we assign them `location_id = -3` or some other value. Let me know what works